pkg/constants/constants.go

```diff
@@ -6,6 +6,16 @@ import (
 	"path/filepath"
 )
 
+type UkiOutput string
+
+const IsoOutput UkiOutput = "iso"
+const ContainerOutput UkiOutput = "container"
+const DefaultOutput UkiOutput = "artifact"
```

Comment:
this is actually what I meant originally (woops)..

we call this const "DefaultOutput" but we refer to "artifact" which is quite generic, but aren't we just generating UKI files and its configs?  

would make sense to me to call this "UKI", so it's more self-explanatory (and you don't have to explain what kind of artifacts are you generating here everywhere)

Reply:
ahh yeah, I gotchu.

So change this into `UKI` ?

Reply:
yep yep, easier to grasp for everyone, less to explain

Reply:
ah yeah now I see what you meant 👍

Reply:
doneeee, no idea why github doesnt say its outdated....